src/pages/winners.js

```diff
@@ -1,35 +1,45 @@
 import React from 'react'
-import '../styles/pages.css'
-import '../styles/colors.css'
-import '../../node_modules/materialize-css/dist/css/materialize.min.css'
+import '../styles/winners.css'
+import '../styles/animated/animated.min.css';
+
+import Matrix from "../components/matrix";
+import Winners from "../components/winners";
+import Footer from '../components/footer.js'
+// 2d Arr for winner list, each row takes 4 entries
```

Comment:
potentially dumb question: does this support more than 4 entry rows?

Reply:
No... but it is a 2d Array, the people will still be in the same section.